### DISCUSSION TITLE
will the old ids be changed

### DISCUSSION THREAD

**Discussion by Dung288-806** [dr5hn/countries-states-cities-database#454] [Q&A]
I'm using JSON files of countries, states, and cities to build my database. I want to ask you a question, when you add new, edit, or delete records, will the old ids be changed?.
For example, My country is VietNam has an id equal to 240. Then you add a new or delete a country. The id of Vietnam will be changed? (241 or 239.. etc).
This is super important information for me! Please let me know.

**Comment by dr5hn** [comment]
@Dung288-806 

- Vietnam id will remain as it is, as this DB is being used in lots of production sites.
- There won't be any deletion of the country unless it's officially announced.
- There won't be any changes of IDs in the states and cities table as well.
- There will be deletion of states and cities if there is a duplicate entry existing in DB or they're officially removed from states.

Note: Official means We obtain this information from official websites like iso.org.
